packages/react-next/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -0,0 +1,302 @@
+import { ICheckboxStyleProps, ICheckboxStyles } from './Checkbox.types';
```

Comment:
i'm not sure we want to fork this file now. I would prefer we start with importing Checkbox.styles from OUFR package

Reply:
Is there some clear guidance on what should be forked, and what imported from OUFR? This whole business of staging a separate package rather than a separate branch is.. novel. 

Is there a plan to clean up all the cross references later on?

Reply:
We are figuring out these as we go :). I doubt next package will ever have no dependency on current

Reply:
We did a branch for 7 and it didn't go so well, hence trying side-by-side this time. That said they both have their issues...if we start on side-by-side and early on don't feel like it's viable we may still be able to switch back.

Reply:
I'm not familiar with the issues your team had with v7, and I'll work within whatever framework you guys implement as long as there's clear guidance. But I fear that this method makes it harder to see what changed in a PR from baseline, and easier to lose changes made to v7 during the v8 development.

Reply:
My intuition is to copy the whole component content and when shipping, copy it back and diff, rather than one-offing specific files.

I recognize there may be concerns.